dotcom-rendering/src/web/components/SecureSignupIframe.importable.tsx

```diff
@@ -329,6 +329,8 @@ export const SecureSignupIframe = ({
 	return (
 		<>
 			<iframe
+				// @TODO check with @dblatcher if there is a better title
+				title={`Sign up to ${newsletterId}`}
```

Comment:
Hey @guardian/newsletters & @dblatcher, is there a better title we could pass to this `iframe`?